deny.toml

```diff
@@ -107,6 +107,7 @@ allow = [
   "OpenSSL",
   "Zlib",
   "Unicode-3.0",
+  "CDLA-Permissive-2.0"
```

Comment:
CI rejected a license, "CDLA-Permissive-2.0" for webpki-root, okayed for this list by @wilwade